typescript/cli/src/logger.ts

```diff
@@ -51,3 +56,6 @@ export const errorRed = (...args: any) => logColor('error', chalk.red, ...args);
 
 // No support for table in pino so print directly to console
 export const logTable = (...args: any) => console.table(...args);
+
+export const indent = (msg: string, indentLevel = 2) =>
```

Comment:
This could probably live somewhere else that is more appropriate

Reply:
please use this instead (also accounts for all lines and not just the first):
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/e38d31685eef7b5b53e2c17879bd3306a34b1443/typescript/cli/src/utils/files.ts#L214